moveit_commander/package.xml

```diff
@@ -28,7 +28,7 @@
   <run_depend>rospy</run_depend>
   <run_depend>sensor_msgs</run_depend>
   <run_depend>shape_msgs</run_depend>
-  <run_depend>tf</run_depend>
```

Comment:
Ah, this removal will cause problems unfortunately. tf2 python bindings don't have the conversions we apparently depend on:
```
~/dev/moveit_dev/src/moveit/moveit_commander$ grep -r tf
src/moveit_commander/conversions.py:import tf
src/moveit_commander/conversions.py:        q = tf.transformations.quaternion_from_euler(pose_list[3], pose_list[4], pose_list[5])
src/moveit_commander/interpreter.py:import tf
src/moveit_commander/interpreter.py:                    q = tf.transformations.quaternion_from_euler(float(clist[4]), float(clist[5]), float(clist[6]))
src/moveit_commander/move_group.py:import tf
src/moveit_commander/move_group.py:                (r, p, y) = tf.transformations.euler_from_quaternion(pose[3:])
package.xml:  <run_depend>tf2</run_depend>

```
If you'd like to leave this change off for now, I can develop a longer-term fix for this python package.

Reply:
Thanks for spotting this! Are you going to revert this?

Reply:
Sure. I'll push that now